README.html

```diff
@@ -3,27 +3,16 @@
 <HEAD>
 	<META HTTP-EQUIV="CONTENT-TYPE" CONTENT="text/html; charset=utf-8">
 	<TITLE>GeoTools @RELEASE@ README</TITLE>
-	<META NAME="GENERATOR" CONTENT="LibreOffice 3.3  (Linux)">
-	<META NAME="CREATED" CONTENT="0;0">
-	<META NAME="CHANGEDBY" CONTENT="Jody Garnett">
-	<META NAME="CHANGED" CONTENT="20110428;21541200">
-	<META NAME="CocoaVersion" CONTENT="1038.11">
-	<META NAME="CHANGEDBY" CONTENT="Jody Garnett">
 	<META HTTP-EQUIV="Content-Style-Type" CONTENT="text/css">
 	<STYLE TYPE="text/css">
 	<!--
-		H1.western { font-family: "Liberation Sans", sans-serif }
-		H2.cjk { font-family: "WenQuanYi Micro Hei" }
-		H2.ctl { font-family: "Lohit Hindi" }
-		H3.western { font-family: "Liberation Sans", sans-serif }
-		H3.cjk { font-family: "WenQuanYi Micro Hei" }
-		H3.ctl { font-family: "Lohit Hindi" }
+		H1 { font-family: "Liberation Sans", sans-serif }
```

Comment:
I do wonder if some of these are important for any non-english text (perhaps links - do we link to translations of the docs here?)?

Reply:
No this is just boilerplate generated by LibreOffice